ui/index.html

```diff
@@ -17,8 +17,7 @@
 
 
 
-  <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
undo

Reply:
I understand that you uses this to get statistics from users access. U 
But it couldn't be done directly on the server side?
When you disagree with a change "undo 😁" will not affect original project but can I still removing it from a forked branch or I'm missing some other functionality that this tag provide?

Reply:
Sure, but first this change would have to be made in the BIPES server and then remove in the codebase. Not the other way around.

Direct answer: Yes, you can remove from your branch, just don't include in the [pull request](https://stackoverflow.com/questions/34027850/how-to-pull-request-a-specific-commit).

Reply:
Sure, but first this change would have to be made in the BIPES server and then remove in the codebase. Not the other way around.

Direct answer: Yes, you can remove from your branch, just don't include in the [pull request](https://stackoverflow.com/questions/34027850/how-to-pull-request-a-specific-commit).